src/library/autodj/dlgautodj.h

```diff
@@ -23,13 +23,14 @@ class DlgAutoDJ : public QWidget, public Ui::DlgAutoDJ, public LibraryView {
               Library* pLibrary,
               AutoDJProcessor* pProcessor, TrackCollection* pTrackCollection,
               KeyboardEventFilter* pKeyboard);
-    virtual ~DlgAutoDJ();
-
-    void onShow();
-    void onSearch(const QString& text);
-    void loadSelectedTrack();
-    void loadSelectedTrackToGroup(QString group, bool play);
-    void moveSelection(int delta);
+    virtual ~DlgAutoDJ() override;
```

Comment:
Actually on reading that link I posted again, it looks like they were talking about `virtual` without `override` (i.e. before `override` existed). Maybe I'll start doing just `override` without the virtual.